### PR TITLE
pythonPackages.furl

### PR DESCRIPTION
**Pull request by vanzef**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/development/python-modules/furl/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{ stdenv, buildPythonPackage, fetchPypi, pythonPackages }:
```
Comment (jonringer) [inline pkgs/development/python-modules/furl/default.nix]: don't import pythonPackages, rather the individual python packages `flake8, six, orderedmultidict`
```suggestion
{ stdenv, buildPythonPackage, fetchPypi, flake8, six, orderedmultidict }:
```

File: pkgs/development/python-modules/furl/default.nix [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+buildPythonPackage rec {
+  pname = "furl";
+  version = "2.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1v2lakx03d5w8954a39ki44xv5mllnq0a0avhxykv9hrzg0yvjpx";
+  };
+
+  nativeBuildInputs = [ pythonPackages.flake8 ];
```
Comment (jonringer) [inline pkgs/development/python-modules/furl/default.nix]: nativeBuildInputs maps to `setup_requires`, if this is for tests, put it in `checkInputs`
```suggestion
  nativeBuildInputs = [ flake8 ];
```

File: pkgs/development/python-modules/furl/default.nix [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+  propagatedBuildInputs = with pythonPackages; [ six orderedmultidict ];
```
Comment (jonringer) [inline pkgs/development/python-modules/furl/default.nix]:
```suggestion
  propagatedBuildInputs = [ six orderedmultidict ];
```

File: pkgs/development/python-modules/orderedmultidict/default.nix [hunk 4]
```diff
@@ -0,0 +1,22 @@
+{ stdenv, buildPythonPackage, fetchPypi, pythonPackages }:
```
Comment (jonringer) [inline pkgs/development/python-modules/orderedmultidict/default.nix]:
```suggestion
{ stdenv, buildPythonPackage, fetchPypi, flake8, six }:
```

File: pkgs/development/python-modules/orderedmultidict/default.nix [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+buildPythonPackage rec {
+  pname = "orderedmultidict";
+  version = "1.0.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1bc2v0yflsxjyyjx4q9wqx0j3bvzcw9z87d5pz4iqac7bsxhn1q4";
+  };
+
+  nativeBuildInputs = [ pythonPackages.flake8 ];
```
Comment (jonringer) [inline pkgs/development/python-modules/orderedmultidict/default.nix]:
```suggestion
  nativeBuildInputs = [ flake8 ];
```

File: pkgs/development/python-modules/orderedmultidict/default.nix [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+  propagatedBuildInputs = with pythonPackages; [ six ];
```
Comment (jonringer) [inline pkgs/development/python-modules/orderedmultidict/default.nix]:
```suggestion
  propagatedBuildInputs = [ six ];
```

**Comment by vanzef** [comment]
@jonringer, thanks for suggestions, I've applied them all.

**Review by jonringer — APPROVED** [review]
`nix-review` passes on NixOS
diff LGTM

```
[7 built, 10 copied (2.5 MiB), 0.5 MiB DL]
https://github.com/NixOS/nixpkgs/pull/66675
4 package were build:
python27Packages.furl python27Packages.orderedmultidict python37Packages.furl python37Packages.orderedmultidict
```

**Comment by risicle** [comment]
`nix-review` is happy for me on macos 10.13.

**Pull request merged**